src/MusicStore/Get-Crossgen.ps1

```diff
@@ -32,9 +32,9 @@ function Get-CoreCLRVersion()
 
     foreach ($name in $json["libraries"].Keys)
```

Comment:
> restore your project for that RID, and grab cross gen from there

Does from there imply that I have to know where NuGet put the package? Or is there another way to do it?